prac_09/band.py

```diff
@@ -0,0 +1,25 @@
+"""
+CP1404/CP5632 Practical - Band.py.
+"""
+
+
+class Band:
+    """Represent class Band."""
+
+    def __init__(self, name):
+        """Initialize Band with name."""
+        self.name = name
+        self.musicians = []
+
+    def __str__(self):
+        """Represent a string of Band object."""
+        musician_info = ''.join([str(musician) for musician in self.musicians])
```

Comment:
the name musician_info represents the string concatenation of all musicians. Although the name is good, it could be more meaningful to the context such as musician_list.

Reply:
musician_info is a bit of a hard name but I think its fine it doesnt just have musicians but also their details, musician_list is actually a worse name as musician_info is not a list it is a string so this is misinformation and we already have a list of musicians so this could add further confusion to the code

though I appreciate the feedback